docs/src/optimization_packages/manopt.md

```diff
@@ -11,3 +11,115 @@ To use the Optimization.jl interface to Manopt, install the OptimizationManopt p
 import Pkg;
 Pkg.add("OptimizationManopt");
 ```
+
+## Methods
+
+The following methods are available for the `OptimizationManopt` package:
+
+  - `GradientDescentOptimizer`: Corresponds to the [`gradient_descent`](https://manoptjl.org/stable/solvers/gradient_descent/) method in Manopt. 
+  - `NelderMeadOptimizer` : Corresponds to the [`NelderMead`](https://manoptjl.org/stable/solvers/NelderMead/) method in Manopt.
+  - `ConjugateGradientDescentOptimizer`: Corresponds to the [`conjugate_gradient_descent`](https://manoptjl.org/stable/solvers/conjugate_gradient_descent/) method in Manopt.
+  - `ParticleSwarmOptimizer`: Corresponds to the [`particle_swarm`](https://manoptjl.org/stable/solvers/particle_swarm/) method in Manopt.
+  - `QuasiNewtonOptimizer`: Corresponds to the [`quasi_Newton`](https://manoptjl.org/stable/solvers/quasi_Newton/) method in Manopt.
+  - `CMAESOptimizer`: Corresponds to the [`cma_es`](https://manoptjl.org/stable/solvers/cma_es/) method in Manopt.
+  - `ConvexBundleOptimizer`: Corresponds to the [`convex_bundle_method`](https://manoptjl.org/stable/solvers/convex_bundle_method/) method in Manopt.
+  - `FrankWolfeOptimizer`: Corresponds to the [`FrankWolfe`](https://manoptjl.org/stable/solvers/FrankWolfe/) method in Manopt.
+  
+The common kwargs `maxiters`, `maxtime` and `abstol` are supported by all the optimizers. Solver specific kwargs from Manopt can be passed to the `solve` 
+function or `OptimizationProblem`.
+
+!!! note
+
+    The `OptimizationProblem` has to be passed the manifold as the `manifold` keyword argument.
+
+## Examples
+
+The Rosenbrock function on the Euclidean manifold can be optimized using the `GradientDescentOptimizer` as follows:
+
+```@example Manopt1
+using Optimization, OptimizationManopt
+rosenbrock(x, p) = (p[1] - x[1])^2 + p[2] * (x[2] - x[1]^2)^2
+x0 = zeros(2)
+p = [1.0, 100.0]
+
+R2 = Euclidean(2)
+
+stepsize = Manopt.ArmijoLinesearch(R2)
+opt = OptimizationManopt.GradientDescentOptimizer()
+
+optf = OptimizationFunction(rosenbrock, Optimization.AutoZygote())
+
+prob = OptimizationProblem(
+    optf, x0, p; manifold = R2, stepsize = stepsize)
+
+sol = Optimization.solve(prob, opt)
+```
+
+<!-- The Karcher mean problem on the SPD manifold with the Frank-Wolfe algorithm can be solved as follows:
```

Comment:
This was not correct. The FW here computed a constrained version of the mean. The mean itself can actually be computed very nicely with GD

Reply:
```suggestion
<!-- The box-constrained Karcher mean problem on the SPD manifold with the Frank-Wolfe algorithm can be solved as follows:
```

Right, this can be more accurate here 👍